docs/rules/space-in-parens.md

```diff
@@ -87,8 +89,21 @@ var foo = ( 1 + 2 ) * 3;
 
 An object literal may be used as a third array item to specify exceptions, with the key `"exceptions"` and an array as the value. These exceptions work in the context of the first option. That is, if `"always"` is set to enforce spacing, then any "exception" will *disallow* spacing. Conversely, if `"never"` is set to disallow spacing, then any "exception" will *enforce* spacing.
 
+Note that this rule only enforces spacing within parentheses; it does not check spacing within curly or square brackets, but will enforce or disallow spacing of those brackets if and only if they are adjacent to an opening or closing parenthesis.
+
 The following exceptions are available: `["{}", "[]", "()", "empty"]`.
 
+### Empty Exception
+
+The empty exception is a special use case that currently has a few unintuitive behavioral quirks:
```

Comment:
I believe it's a good thing to have this small section that clarifies how the rule works with empty parens at the moment. 

But I'd be fine with removing this sentence and leaving just the list. 

'currently' might imply that this is going to be changed in the near future (which isn't planned and accepted at the moment), and 'unintuitive' is subjective.

Reply:
Roger that